prow/repoowners/repoowners.go

```diff
@@ -371,28 +371,27 @@ func (o *RepoOwners) walkFunc(path string, info os.FileInfo, err error) error {
 		return nil
 	}
 
-	b, err := ioutil.ReadFile(path)
```

Comment:
Please don't include this in the parse functions, instead read the file once and pass the bytes to the parse functions. This is important to avoid unnecessarily reading every OWNERS file with the `FullConfig` format twice.